src/Lean/Server/FileWorker.lean

```diff
@@ -134,7 +134,9 @@ section Elab
       -- This will overwrite existing ilean info for the file since this has a
       -- higher version number.
       publishIleanInfoUpdate m ctx.hOut snaps
```

Comment:
We might want to move this behind the sleep as well, though it doesn't look like it currently compares document versions to deal with racing updates.

Reply:
Hmm, I don't think the current version is incorrect though.  It's merely redundant notifications.